src/server/pfs/cmds/cmds.go

```diff
@@ -771,6 +771,27 @@ The squash will fail if it includes a commit with no children`,
 	shell.RegisterCompletionFunc(squashCommit, shell.BranchCompletion)
 	commands = append(commands, cmdutil.CreateAlias(squashCommit, "squash commit"))
 
+	deleteCommit := &cobra.Command{
+		Use:   "{{alias}} <commit-id>",
+		Short: "Delete the sub-commits of a commit.",
+		Long: `Delete the sub-commits of a commit.  The data in the sub-commits will be lost.
+This operation is only supported if the commit has no children.`,
```

Comment:
I guess more precisely

```suggestion
This operation is only supported if none of the sub-commits have children.`,
```